base/base/EnumReflection.h

```diff
@@ -1,5 +1,8 @@
 #pragma once
 
+/// To cover ZooKeeperConstants -> See contrib/magic_enum/doc/limitations.md#enum-range
+#define MAGIC_ENUM_RANGE_MIN -100
+#define MAGIC_ENUM_RANGE_MAX 1000
```

Comment:
None of these two is ever actually used, right?

Reply:
They are for magic_enum.hpp -> https://github.com/Neargye/magic_enum/blob/master/doc/limitations.md#enum-range. I just linked the internal file instead of the github url because I wanted to be fancy